fvtest/compilertriltest/LinkageTest.cpp

```diff
@@ -61,8 +61,10 @@ TYPED_TEST(LinkageTest, InvalidLinkageTest) {
     SKIP_ON_PPC(MissingImplementation) << "Test is skipped on POWER because calls are not currently supported (see issue #1645)";
     SKIP_ON_PPC64(MissingImplementation) << "Test is skipped on POWER 64 because calls are not currently supported (see issue #1645)";
     SKIP_ON_PPC64LE(MissingImplementation) << "Test is skipped on POWER 64le because calls are not currently supported (see issue #1645)";
-    SKIP_ON_S390(MissingImplementation) << "Test is skipped on S390 because calls are not currently supported (see issue #1645)";
-    SKIP_ON_S390X(MissingImplementation) << "Test is skipped on S390x because calls are not currently supported (see issue #1645)";
+    if ("z/OS" != omrsysinfo_get_OS_type()) {
+       SKIP_ON_S390(MissingImplementation) << "Test is skipped on S390 because calls are not currently supported (see issue #1645)";
+       SKIP_ON_S390X(MissingImplementation) << "Test is skipped on S390x because calls are not currently supported (see issue #1645)";
+    }
```

Comment:
It seems unfortunate to have to use an if statement. Would it be worth implementing a macro like:

```c++
SKIP_ON_S390X_ZOS(MissingImplementation) << "...";
```

Reply:
Unfortunate indeed. No better time to fix it than now though. Added new macros and made use of them in 13a5b43